diff --git a/node_modules/@types/react-vertical-timeline-component/index.d.ts b/node_modules/@types/react-vertical-timeline-component/index.d.ts
index a4b8cd2..e0a9d82 100644
--- a/node_modules/@types/react-vertical-timeline-component/index.d.ts
+++ b/node_modules/@types/react-vertical-timeline-component/index.d.ts
@@ -11,7 +11,7 @@ export interface VerticalTimelineProps {
     animate?: boolean | undefined;
     children?: React.ReactNode;
     className?: string | undefined;
-    layout?: '1-column' | '2-columns' | undefined;
+    layout?: '1-column' | '1-column-left' | '1-column-right' | '2-columns' | undefined;
     lineColor?: string | undefined;
 }
 
